docs/how-to-install-kubectl-helm.md

```diff
@@ -0,0 +1,448 @@
+This page shows you how to install kubectl and helm on macOS, Linux (Ubuntu) and Windows
+
+___
+
+# Menu
+
+- Install `kubectl` for each OS you need:
+	+ [Mac Os](#kubect_mac_os)
+	+ [Linux(Ubuntu)](#kubectl_linux)
+	+ [Windows](#kubectl_windows)
+
+- Install `helm` for each OS you need:
+	+ [Mac Os](#helm_mac_os)
+	+ [Linux(Ubuntu)](#helm_linux)
+	+ [Windows](#helm_windows)
+
+# Kubectl
+``kubectl`` is the Kubernetes command line tool, which can be used to deploy settings to the cluster.
+In this section, you should know how to install kubectl.
+
+## <a name="kubect_mac_os"></a>Mac Os
+
+### Install
+
+- Latest version:

```

Comment:
If we mention 2 ways to install, the structure should be 
- Install Kubectl with brew
- Install Kubectl with curl

And remove the unnecessary lines.